x-pack/plugins/security/server/authentication/providers/pki.test.ts

```diff
@@ -0,0 +1,534 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PeerCertificate, TLSSocket } from 'tls';
```

Comment:
__note:__ hmm, will place this import after `jest.mock` calls. Not sure why it's here.